src/components/auth/Auth.css

```diff
@@ -1,102 +1,104 @@
-
```

Comment:
More Prettier formatting... probably not going to leave comments for each of these 🤷‍♂